src/content/docs/highlights/v0.25.md

```diff
@@ -0,0 +1,385 @@
+---
+title: v0.25.0
+---
+
+<https://github.com/ratatui-org/ratatui/releases/tag/v0.25.0>
+
+⚠️ See the [breaking changes](https://github.com/ratatui-org/ratatui/blob/main/BREAKING-CHANGES.md)
+for this release.
+
+## Ratatui Website 📚
+
+We revamped our entire website ([ratatui.rs](https://ratatui.rs)) with [Astro](https://astro.build/)
+and updated the documentation/tutorials.
+
+![image](https://github.com/ratatui-org/website/assets/24392180/bdff62e6-89d8-4d4e-9e25-d0652edffde4)
+
+Here are some of the sections that we recommend checking out:
+
+- [Tutorials](https://ratatui.rs/tutorials/)
+- [Concepts](https://ratatui.rs/concepts/)
+- [How-to](https://ratatui.rs/how-to/)
+- [Apps/Widgets Showcase](https://ratatui.rs/showcase/)
+
+All the code is available at <https://github.com/ratatui-org/website>
+
+Check out the [good first issues here](https://github.com/ratatui-org/website/issues) if you are
+interested in contributing to our website!
+
+---
+
+## Awesome Ratatui 🦀
+
+We created an [awesome list](https://github.com/sindresorhus/awesome) for curating the awesome
+applications/libraries built with `ratatui`!
+
+See: <https://github.com/ratatui-org/awesome-ratatui>
+
+If you have something built with/for Ratatui, feel free to
+[add it to this list](https://github.com/ratatui-org/awesome-ratatui/blob/main/contributing.md)!
+
+---
+
+## Ratatui Templates 📝
+
+We gathered the available `ratatui` templates under a single repository:
+<https://github.com/ratatui-org/templates>
+
+Currently, we have two templates:
+
+- Simple: A template for bootstrapping a simply `ratatui` application quickly.
+- Async: A more advanced template which uses crates such as `tokio` and `tracing` for creating an
+  async `ratatui` application.
+
+This repository is still under construction so feel free to take a look and help us out.
+Additionally, we really appreciate if you have another template and interested in contributing it.
+
+---
+
+## New/Updated Examples 🆕
+
+We added a new example
+([`ratatui-logo`](https://github.com/ratatui-org/ratatui/blob/main/examples/ratatui-logo.rs)) which
+shows how to render a big text using half blocks.
+
+![ratatui-logo](https://raw.githubusercontent.com/ratatui-org/ratatui/images/examples/ratatui-logo.gif)
```

Comment:
Not sure if it is better to add the link to the demo (as below) or add GIF directly here.